lib/fun_with_flags/store/persistent/in_memory.ex

```diff
@@ -0,0 +1,163 @@
+defmodule FunWithFlags.Store.Persistent.InMemory do
+  alias FunWithFlags.{Config, Gate, Flag, Store.Persistent.Ecto.Record}
```

Comment:
If this is an in-memory adapter, then it shouldn't use a module scoped under `...Ecto`. That could even not be available in theory, [because](https://github.com/tompave/fun_with_flags/blob/v1.4.0/lib/fun_with_flags/store/persistent/ecto/record.ex#L1-L3):

```elixir
if Code.ensure_loaded?(Ecto) do
  defmodule FunWithFlags.Store.Persistent.Ecto.Record do
```

Reply:
yup, I was actually investigating this around the same time I opened this PR.
We had our own fork of FWF precisely to remove those checks, because we were having occasional compilation issues.

On further inspection, I think it's because I was sometimes doing `mix deps.compile --force fun_with_flags`. But I didn't know about the `--include-children`. So FWF would end up not being compiled with ecto support.